### DISCUSSION TITLE
Unable to use the application without downloading the supplementary app

### DISCUSSION THREAD

**Discussion by feliceconfuso** [aclap-dev/video-downloadhelper#819] [Help and Q&A]
### Your question

I find the app absolutely brilliant, however I can't use it because before using it it asks me to download the **coapp-v2 supplementary app**. But since I don't have the privileges to install it (I'm just a "user" on my win11) I'm stuck. Is there a way to solve it?

### Addon & CoApp version and details

I can't use the app

**Comment by GrampaWildWilly** [comment]
There must be an administrator user on your system.  Use it to upgrade your ordinary user to an administrator.  If you need more help, https://www.elevenforum.com/ is a wonderful resource.

**Reply by feliceconfuso** [reply]
I don't think I understood correctly. Is there a way I can have administrator privileges without knowing the password?
Normally there are versions of "portable" applications that do not require administrator privileges... Is there such a thing?

**Reply by GrampaWildWilly** [reply]
I'm not really sure how to deal with this situation.  This is your computer, right?  This isn't a computer that belongs to your employer or anything like that, is it?  When you install W11, you have to create users.  The initial user has administrator privileges.  I've always just signed on with an administrator user.  If you can't figure this out with help from https://www.elevenforum.com/, I don't think I can help you.  This isn't so much a VDH issue as a Windows 11 usage issue, thus entirely off topic for this forum.  You need to make an effort to find help on elevenforum.

It occurs to me that you bought your system with W11 pre-installed.  In that case, you need to talk to whatever store you bought it from.  They have to help you create an administrator user that you can log in on, that you know the password for.  It is even possible to create an administrator user that has no password.  That's how I run my system.  But nobody but me has access to my computer.  You need to configure your system as is appropriate for your environment.

**Reply by feliceconfuso** [reply]
ok, thank's

**Reply by GrampaWildWilly** [reply]
Do come back eventually & tell us how you've gotten on.  Explain whatever you did to get things squared away.  It might help others some day.
